sensor_server:
  pc_pub_topic: pc_pub
  pc_sub_topic: /photoneo_center/sensor/points
  img_sub_topic: /photoneo_center/sensor/image_color
  cam_sub_topic: /photoneo_center/sensor/camera_info
  sensor_service_name: sensor_service
  sensor_pc2_service_name: sensor_pc2_service

  
cloud_process:
  planar_threshold: 0.01
  x_min: -1.1
  y_min: -1.1
  z_min: 0.01
  x_max: 1.1
  y_max: 1.1
  z_max: 1.01
